Dockerfile

```diff
@@ -90,3 +90,7 @@ COPY --link --from=release /home/app/src/playbook-website/public /home/app/src/p
 COPY --link --from=release /home/app/src/node_modules/@powerhome/playbook-icons/icons /home/app/src/temp-icons
 RUN cp -r /home/app/src/temp-icons/* /home/app/src/playbook-website/app/javascript/images/
 RUN rm -rf /home/app/src/temp-icons
+
+COPY --link --from=release /home/app/src/node_modules/@powerhome/playbook-icons/aliases.json /home/app/src/aliases.json
```

Comment:
We should avoid using docker here. 
Could we just initialize the file in an initializer? 

```.rb
# config/initializers/load_json.rb

require 'json'

module_path = Rails.root.join('node_modules', 'playbook-icons', 'aliases.json')
JSON_DATA = JSON.parse(File.read(module_path))
```

Reply:
And then the importing could look like this (if we need the import in a JS file)
```
// app/javascript/packs/application.js

import jsonData from 'my-module/data.json';

console.log(jsonData);
```